build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
     implementation 'org.flywaydb:flyway-core'
     implementation 'org.mapstruct:mapstruct:1.5.2.Final'
     implementation 'com.google.guava:guava:31.1-jre'
+    testImplementation 'org.projectlombok:lombok:1.18.22'
```

Comment:
move this line together with other `testImplementation`

Reply:
remove it